plugins/sandbox/controller.go

```diff
@@ -308,3 +335,84 @@ func (c *controllerLocal) getSandbox(ctx context.Context, id string) (runtimeAPI
 
 	return sandbox.NewClient(shim.Client())
 }
+
+func (c *controllerLocal) loadExistingSandboxes(ctx context.Context) error {
```

Comment:
I'm not sure if that's the right place for this logic. Can we push this down to manager? (same for bundles above)

Reply:
done